pytket/extensions/azure/backends/azure.py

```diff
@@ -87,10 +119,24 @@ def _get_workspace(
     OpType.SetBits,
     OpType.CopyBits,
     OpType.ClExpr,
-    OpType.WASM,
 }
 
 
+_QUANTINUUM_TARGET_GATESET = {
```

Comment:
If we reduce this to `ZZPhase`, `Rx` and `Rz` we can be more confident that the final circuit won't be significantly worse in terms of single-qubit gates.

Reply:
Do we want to do that for all optimisation level? This list is only relevant for level 0, the other level are compiling to ZZPhase, rx and rz.

Reply:
Didn't spot that -- that's reasonable then.